### ISSUE TITLE
Steam boiler and blast furnace input/output pipes disconnect after server restart.

### ISSUE THREAD

**Issue by Ivolutio** [railcraft-reborn/railcraft#261]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description of the Bug

Pipes to insert/extract to and from (multiblock?) machines disconnect after server restart.
I have tested encountered this with the Steam Boiler and Blast Furnace from Railcraft Reborn, not any other mods.
I have tried using both Mekanism pipes and Modern Dynamics pipes. I tried placing a block next to them to possibly give them an update, but it doesnt change.
I am playing the Direwolf20 1.21 modpack.

### Railcraft Reborn version

1.2.2

### NeoForge/Forge version

21.1.69

### Minecraft version

1.21

### Minecraft server

Yes

### To Reproduce

1. Connect Mekanism or Modern Dynamics pipes to steam boiler or the blast furnace.
2. Configure them to output/extract or insert.
3. Restart minecraft server
4. log back in and they are disconnected.

### Expected behavior

Pipes should stay connected after server restart.

### Screenshots & Video

Before server restart
![afbeelding](https://github.com/user-attachments/assets/2ad124a6-1f41-412f-821d-723b544f2e6a)
![javaw_FermtQczQj](https://github.com/user-attachments/assets/2a3639f1-430a-410e-8058-6397f01c2fd3)

After server restart
![afbeelding](https://github.com/user-attachments/assets/f072a832-ed3b-4305-bbae-643840197f3d)
![javaw_T3vj0Zjjkj](https://github.com/user-attachments/assets/f04e2008-f862-4e5c-a586-e06d66a7a333)

**Comment by T1ckss** [comment]
Yup its related to #253 
Issue seems to be the "entity" , that makes up the blast furnace , coke oven , turbine , iron tanks isn't formed right after a server startup and waits till the server "ticks"

While pipes and other blocks from other mods checks when they are loaded into the world if the "entity" they are connecting to has capability and disconnects if they are not.

The solution is when those entity check if they have inventory is to create the "entities" than and there instead of waiting for a server tick.

One work around is to use railcraft loaders and unloaders as a go between bridge.

**Comment by Ivolutio** [comment]
@T1ckss Thanks for your reply.
> One work around is to use railcraft loaders and unloaders as a go between bridge.

I am confused how to achieve this, I thought the loaders/unloaders only input/output from carts? Or can you configure them to output to nearby containers somehow?

**Comment by T1ckss** [comment]
> @T1ckss Thanks for your reply.
> 
> > One work around is to use railcraft loaders and unloaders as a go between bridge.
> 
> I am confused how to achieve this, I thought the loaders/unloaders only input/output from carts? Or can you configure them to output to nearby containers somehow?

They naturally will unload/load from any of their sides, just kinda have to think "opposite" to their names.

**Comment by ArtpokeBr** [comment]
Yup, can also confirm that this still a problem, as reported [here](https://github.com/FTBTeam/FTB-Modpack-Issues/issues/6400)
